src/Core/src/Platform/Android/NativeTicker.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Animation;
+using Android.Content;
+using Android.OS;
+using Microsoft.Maui;
+namespace Microsoft.Maui.Animations
+{
+	public class NativeTicker : Ticker
+	{
+		static List<NativeTicker> tickers = new List<NativeTicker>();
+		public static void CheckPowerSaveModeStatus()
+		{
+			foreach (var t in tickers)
+				t.checkPowerSaveModeStatus();
+		}
+		readonly IMauiContext MauiContext;
+		ValueAnimator _val;
+		bool _systemEnabled;
+		public NativeTicker(IMauiContext mauiContext)
+		{
+			MauiContext = mauiContext;
+			_val = new ValueAnimator();
+			_val.SetIntValues(0, 100); // avoid crash
+			_val.RepeatCount = ValueAnimator.Infinite;
+			_val.Update += (s, e) => Fire?.Invoke();
+			;
+			checkPowerSaveModeStatus();
+			tickers.Add(this);
+		}
+		~NativeTicker()
+		{
+			tickers.Remove(this);
```

Comment:
Not sure if this is the best place to remove this object from the list. 
AFAIK We can't tell when finalizers will be called, maybe the GC doesn't call it at all, also implementing the destructor makes the object lives more generations on GC. 

On top of my head I believe that good approaches could be:
1. Implementing the dispose pattern
2. Making the `List<NativeTicker>` to be a `List<WeakReference<NativeTicker>>`

Reply:
Probably true, however I did a list for completeness. But there will only ever be one.